homework_postgres/PostgreSQL-Students.ipynb

```diff
@@ -40,9 +40,27 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.                                    (10, 'Lana Wachowski', 1965);
</pre><p>Why field born is TEXT  but interger value is still insertable and not raise error?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/haicheviet/dataeng-course-1/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='162'/>